docs/en/operation/Journal.md

```diff
@@ -158,6 +159,27 @@ the default backup directory would be `hdfs://192.168.1.1:9000/alluxio_backups`.
 The files to retain in the backup directory is limited by `alluxio.master.daily.backup.files.retained`.
 Users can set this property to the number of backup files they want to keep in the backup directory.
 
+### Backup delegation on HA cluster
+Alluxio supports taking backup without causing service unavailability on a HA cluster configuration.
+When enabled, Alluxio leader master delegates backups to standby masters in cluster.
```

Comment:
I just find that in our doc convention, we use leading master https://docs.alluxio.io/os/user/edge/en/contributor/Documentation-Conventions.html.